liquidcore/home/views.py

```diff
@@ -25,7 +25,8 @@ def profile(request):
     return JsonResponse({
         'id': user.get_username(),
         'login': user.get_username(),
-        'email': user.email or user.get_username() + '@' + settings.LIQUID_DOMAIN,
+        'email': user.email or
```

Comment:
Use a parantheses or `\` here to force the indentation of the following line. Even better, extract a variable just above `return` statement.